a.core_concepts.md

```diff
@@ -324,8 +315,13 @@ kubectl exec -it nginx -- env
 # or
 kubectl describe po nginx | grep val1
 # or
-kubectl run nginx --restart=Never --image=nginx --env=var1=val1 -it --rm -- env
+kubectl get po nginx -o jsonpath='{.spec.containers[].env}{"\n"}'
+[map[name:var1 value:val1]] # not pretty but precise
```

Comment:
Add a comment that this will appear on display, otherwise people might try to type that :)